src/includes/performance/custom-performance-metrics/javascript.mdx

```diff
@@ -1,14 +1,16 @@
-Supported in Sentry's JavaScript SDK version `7.0.0` and above.
+Adding custom metrics is supported in Sentry's JavaScript SDK version `7.0.0` and above.
```

Comment:
@lobsterkatie - Please confirm I haven't changed the meaning here.

Reply:
Nah, I think let's leave it above the snippet so they don't start copying and pasting without knowing if they can use it.